frappe/patches/v14_0/copy_mail_data.py

```diff
@@ -21,6 +23,6 @@ def execute():
 			doc.save()
 		EmailFlagQueue = frappe.qb.DocType("Email Flag Queue")
 		frappe.qb.update(EmailFlagQueue) \
-			.set(EmailFlagQueue.imap_folder, "INBOX") \
+			.set(EmailFlagQueue.folder_name, "INBOX") \
```

Comment:
This patch fixes existing queue, what about new records?

https://github.com/frappe/frappe/blob/72fa9a3c1980cbaf29d307928382a5c466a4aa51/frappe/email/inbox.py#L87-L94 

I am unsure of this. @leela can you review this part once?

Reply:
The code has been removed... I don't see `folder_name` in `Email Flag Queue` doctype... 
It seems that part of the code is irrelevant now.

Also, `folder_name` should not be necessary to update the "read" status of an email... **UID** should be enough.